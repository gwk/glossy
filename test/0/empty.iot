{
  'args': ['-test',
    'a b c',
    'abc def',
  ],
}
