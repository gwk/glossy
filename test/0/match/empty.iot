{
  'cmd': 'legs',
  'args': ['test/0/empty.legs', '-match',
    '',
    'a',
    'ab',
  ],
}
