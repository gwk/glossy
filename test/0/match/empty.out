match: '' -- <none>
match: 'a' -> invalid
match: 'ab' -> invalid
